README.md

```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+### Usage Requirements
+
+Please see the [Setup Process](#setup-process) section
+
+### Development Requirements
```

Comment:
The information hierarchy is off here, Setup/Getting started is universal and much more important than Development requirements. Consider moving them below or to `contributing.md`

Reply:
Moved development requirements inside contributing.md file